src/sounds.cpp

```diff
@@ -606,6 +607,8 @@ void sfx::do_ambient()
             case WEATHER_SUNNY:
             case WEATHER_CLOUDY:
             case WEATHER_SNOWSTORM:
+                play_ambient_variant_sound( "environment", "WEATHER_SNOWSTORM", heard_volume, 20,
```

Comment:
I believe `WEATHER_SNOWSTORM` would now play for sunny,cloudy and snowstorm weather. That's not your totally your fault though - previously `WEATHER_SNOW` would now play for sunny, cloudy and snow weather.